packages/controller-utils/src/constants.ts

```diff
@@ -134,6 +134,7 @@ export enum ApprovalType {
   SnapDialogAlert = 'snap_dialog:alert',
   SnapDialogConfirmation = 'snap_dialog:confirmation',
   SnapDialogPrompt = 'snap_dialog:prompt',
+  SnapDialogDefault = 'snap_dialog',
```

Comment:
```suggestion
  SnapDialogDefault = 'snap_dialog:default',
```

Maybe?

Reply:
We chose to use `snap_dialog` without any suffix for the new default: https://github.com/MetaMask/snaps/blob/fbe220572c53810722c2339bd46666b1af71c178/packages/snaps-rpc-methods/src/restricted/dialog.ts#L49

Reply:
No, the constant comes from already merged https://github.com/MetaMask/metamask-extension/pull/26304#discussion_r1701658713